Private/Invoke-Remediation.ps1

```diff
@@ -0,0 +1,273 @@
+function Invoke-Remediation {
+    <#
+    .SYNOPSIS
+    Runs any remediation scripts available.
+
+    .DESCRIPTION
+    This function offers to run any remediation code associated with identified issues.
+
+    .PARAMETER AuditingIssues
+    A PS Object containing all necessary information about auditing issues.
+
+    .PARAMETER ESC1
+    A PS Object containing all necessary information about ESC1 issues.
+
+    .PARAMETER ESC2
+    A PS Object containing all necessary information about ESC2 issues.
+
+    .PARAMETER ESC3
+    A PS Object containing all necessary information about ESC3 issues.
+
+    .PARAMETER ESC4
+    A PS Object containing all necessary information about ESC4 issues.
+
+    .PARAMETER ESC5
+    A PS Object containing all necessary information about ESC5 issues.
+
+    .PARAMETER ESC6
+    A PS Object containing all necessary information about ESC6 issues.
+
+    .INPUTS
+    PS Objects
+
+    .OUTPUTS
+    Console output
+    #>
+
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $false)]
+        $AuditingIssues,
+        [Parameter(Mandatory = $false)]
+        $ESC1,
+        [Parameter(Mandatory = $false)]
+        $ESC2,
+        [Parameter(Mandatory = $false)]
+        $ESC3,
+        [Parameter(Mandatory = $false)]
+        $ESC4,
+        [Parameter(Mandatory = $false)]
+        $ESC5,
+        [Parameter(Mandatory = $false)]
+        $ESC6
+    )
+
+    Write-Host "`nExecuting Mode 4 - Attempting to fix identified issues!`n" -ForegroundColor Green
```

Comment:
Would be ideal to specify a contrasting background color in case the op is running with a light-mode theme ;-) or if they have color blindness and cannot see enough contrast between green and other colors.

Reply:
You are completely correct. I will create an accessibility branch to handle backgrounds for anything with a -ForegroundColor.